controller/array_action/array_mediator_interface.py

```diff
@@ -151,13 +151,14 @@ def get_volume_mappings(self, volume_id):
         raise NotImplementedError
 
     @abstractmethod
-    def map_volume(self, volume_id, host_name):
+    def map_volume(self, volume_id, host_name, connectivity_type):
         """
         This function will find the next available lun for the host and map the volume to it.
 
         Args:
            volume_id : the volume WWN.
            host_name : the name of the host to map the volume to.
+           connectivity_type : connectivity type (iscsi, fc or nvme)
```

Comment:
I would avoid listing examples in a way which sounds exhaustive. consider something like:
```suggestion
           connectivity_type : connectivity type (e.g. iscsi, fc, etc...)
```

Reply:
those are not examples, they are the only options we support.

Reply:
I don't think we should maintain all the options we support here